fabric/src/main/java/io/github/foundationgames/automobility/fabric/controller/controlify/ControlifyInit.java

```diff
@@ -0,0 +1,64 @@
+package io.github.foundationgames.automobility.fabric.controller.controlify;
+
+import dev.isxander.controlify.api.ControlifyApi;
+import dev.isxander.controlify.api.bind.ControlifyBindingsApi;
+import dev.isxander.controlify.api.entrypoint.ControlifyEntrypoint;
+import dev.isxander.controlify.api.event.ControlifyEvents;
+import dev.isxander.controlify.api.guide.ActionPriority;
+import dev.isxander.controlify.api.ingameguide.ActionLocation;
+import dev.isxander.controlify.bindings.BindContext;
+import dev.isxander.controlify.bindings.GamepadBinds;
+import io.github.foundationgames.automobility.Automobility;
+import io.github.foundationgames.automobility.entity.AutomobileEntity;
+import net.minecraft.network.chat.Component;
+
+import java.util.Optional;
+import java.util.Set;
+
+public class ControlifyInit implements ControlifyEntrypoint {
```

Comment:
I usually name my compatibility entrypoints something like ControlifyCompat or AutomobilityControlify, would require the requested change above to rename `ControllerCompat` and corresponding implementations